dndwiki/races/race-concubus.xml

```diff
@@ -1,98 +0,0 @@
-<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
is there a reason we just straight up deleted concubus?

Reply:
file was obsoleted before, because it's a copy of Succubus just gender-neutral.

Reply:
ah that makes sense